docs/.vuepress/public/presets/toml/no-empty-icons.toml

```
@@ -0,0 +1,150 @@
+[buf]
+format = '(with [$symbol($version )]($style))'
```

`$version` doesn't need to be conditional with this preset.
```suggestion
format = '(with [$symbol$version ]($style))'
```

---

Because the whole thing is conditional?